### PR TITLE
fixes-for-customer-name-twice-desktop

### PR DESCRIPTION
**Pull request by priti2jcommerce**
### Description (*)

1. Magento version 2.3 luma theme
2. Desktop view
### Fixed Issues (if relevant)

1. magento/magento2#20830
2. ...

### Manual testing scenarios (*)

1. Go to login page after login ,customer full name is present twice one in welcome message and second in drop down

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by shikhamis11** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for your request. I'm working on Magento instance for you

**Comment by priti2jcommerce** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for your request. I'm working on Magento 2.3-develop instance for you

File: app/code/Magento/Theme/view/frontend/templates/html/header.phtml [hunk 1]
```diff
@@ -15,7 +15,8 @@ $welcomeMessage = $block->getWelcome();
     case 'welcome': ?>
         <li class="greet welcome" data-bind="scope: 'customer'">
             <!-- ko if: customer().fullname  -->
-            <span class="logged-in" data-bind="text: new String('<?= $block->escapeHtml(__('Welcome, %1!', '%1')) ?>').replace('%1', customer().fullname)">
+            <span><?= $block->escapeHtml(__('Welcome,')) ?></span>
```
Comment (orlangur) [inline app/code/Magento/Theme/view/frontend/templates/html/header.phtml]: This is not acceptable, translatable message should retain the same.

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Theme/view/frontend/templates/html/header.phtml (hunk 1 above)
Comment (priti2jcommerce) [reply app/code/Magento/Theme/view/frontend/templates/html/header.phtml]: Hi @orlangur ,

Thanks for the update, made the changes for this.

**Comment by priti2jcommerce** [comment]
Hi @orlangur ,

Any update on this PR?

Thanks,
Priti

File: app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less [hunk 2]
```diff
@@ -525,6 +525,9 @@
 
         .customer-name {
             cursor: pointer;
+            span {
+                display: none;
```
Comment (orlangur) [inline app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less]: Why do we need to hide it? Simply eliminate redundant text rendering

**Review by orlangur — CHANGES_REQUESTED** [review]

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=20832) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=20832) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: priti2jcommerce<br/>:x: priti<hr/>**priti** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less (hunk 2 above)
Comment (priti2jcommerce) [reply app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less]: made the changes for above

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-4666](https://jira.corp.magento.com/browse/ENGCOM-4666) has been created to process this Pull Request

**Comment by priti2jcommerce** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce, here is your [Magento instance](https://i-20832-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-20832-2-3-develop.instances.magento-community.engineering/admin](https://i-20832-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by VasylShvorak** [comment]
:heavy_check_mark: QA passed

**Comment by iGerchak** [comment]
@orlangur, it's incorrect fix because if we remove the "Welcome" block, customers will see just drop-down arrow without any text
Example:
Capture 2019-07-18 at 17 05 47

**Comment by orlangur** [comment]
@iGerchak please report a new issue referring here.

**Pull request merged**